src/lib/phase.mli

```diff
@@ -4,12 +4,17 @@
 (* Type definitions *)
 (* ************************************************************************* *)
 
-type id = Id.t
+type id = Id.t [@@deriving yojson]
 (** Ids for phases *)
 
 type t
 (** Competitions Phases: competions are made of different rounds (prelims,
-    finals, etc..), and each pair (competition * round) is a phase. *)
+    finals, etc..), and each pair (competition * round) is a phase.
+    This type describe a phase of a competition.
+    Only one phase of a specific round is allowed per competition.
+    This table is the reference for the list of judges, the head judge,
```

Comment:
This is not a table. Additionally, this last sentence is also false, since the list of judges is stored elsewhere.